assets/scss/base/djaodjin-dashboard.scss

```diff
@@ -103,9 +103,10 @@ $dashboard-sidebar-color:    white;
 
 .dashboard-nav-main {
     float: left;
-    min-height: 100%;
-    padding-bottom: $dashboard-nav-footer-height; /* must be equal to .dashboard-nav-footer height
-                             in pixel */
+    min-height: 100vh;
```

Comment:
Apparently this change prevents the menu items in the sidebar to scroll when the sidebar content is longer than the browser window height.
Ex: cannot get to the "Rules" menuitem in the following use case:
![Screen Shot 2019-08-01 at 9 27 51 AM](https://user-images.githubusercontent.com/3732869/62310586-afaf8b00-b43e-11e9-8726-4acf3caae436.png)